server/README.md

```diff
@@ -59,3 +59,17 @@ You probably have qbittorrent auto starting when your computer boots up right. T
 approach. I suggest looking up how to create a service to run in your OS whenever it boots up to
 run `npm run server-start` in the root of this repo. I'm not sure I could solve this within this project :/ Ideas are
 welcome
+
```

Comment:
I can do it later, but since there's no .env .env.default anymore, the "Configuration" instructions need to change right. Those settings live in `local_env.sh` now

Reply:
Also, docker is able to auto start containers on boot right? I always assumed so but never really used Docker extensively

Reply:
if the docker-compose is registered with the docker system on the host, then yes.  I am not sure about containers that have been started with `docker container run`, since I always default to docker-compose.  you can also configure with the `restart` config option, if the docker-container itself dies, and you want it to restart itself.